frontend/packages/insights-plugin/__tests__/insights.fixtures.ts

```diff
@@ -0,0 +1,108 @@
+import { PrometheusHealthPopupProps } from '@console/plugin-sdk';
+
+const funcs = () => ({
```

Comment:
`funcs` can not be used for the InsightsPopup test (therefore, they are not needed at all). See props (k8sResult, responses) given to this component.